src/server/server_family.cc

```diff
@@ -499,8 +508,10 @@ struct AggregateLoadResult {
 // It starts one more fiber that waits for all load fibers to finish and returns the first
 // error (if any occured) with a future.
 Future<std::error_code> ServerFamily::Load(const std::string& load_path) {
-  io::Result<std::vector<std::string>> paths_result = snapshot_storage_->LoadPaths(load_path);
+  auto paths_result = snapshot_storage_->LoadPaths(load_path);
   if (!paths_result) {
+    LOG(ERROR) << "Failed to load snapshot: " << paths_result.error().Format();
+
     Promise<std::error_code> ec_promise;
     ec_promise.set_value(paths_result.error());
     return ec_promise.get_future();
```

Comment:
It's a generic error, right? Unfortunately it can be the case, that if created just with a string, its error_code will be a default-value (not set), so the result of Load will actually return that no error happened. You do so for example with "Invalid S3 Path", so I assume its a possible bug

Reply:
Oops - looks like the `Load()` returned error is currently ignored - though could update that to return a `GenericError` too?